chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -936,6 +936,29 @@ impl PeerManagerActor {
             .collect::<Vec<_>>()
     }
 
+    // Get peers that are potentially unreliable and we should avoid routing messages through them.
+    // Currently we're picking the peers that are NOT the highest peers.
+    fn unreliable_peers(&self) -> HashSet<PeerId> {
```

Comment:
this feels like a near copy of highest_height_peers? can we dedupe?

Reply:
I was thinking about it - but the best I came up with was moving the computation of the 'max' to the separate function (which didn't buy much).

Reply:
yeah - and actually after discussion witn you, I plan to change this logic here.